Jint/Native/Json/JsonSerializer.cs

```diff
@@ -303,63 +301,110 @@ private static bool CanSerializesAsArray(ObjectInstance value)
         /// <summary>
         /// https://tc39.es/ecma262/#sec-quotejsonstring
         /// </summary>
-        private static void QuoteJSONString(string value, ref SerializerState target)
+#if NETCOREAPP1_0_OR_GREATER
+        [MethodImpl(MethodImplOptions.AggressiveInlining | MethodImplOptions.AggressiveOptimization)]
```

Comment:
might be easier to just use value `512` without `#if` here as it's (I think) a common practice, casting it to `MethodImplOptions` should be safe